posthog/hogql/modifiers.py

```diff
@@ -51,6 +52,9 @@ def set_default_modifier_values(modifiers: HogQLQueryModifiers, team: "Team"):
     if modifiers.bounceRatePageViewMode is None:
         modifiers.bounceRatePageViewMode = BounceRatePageViewMode.COUNT_PAGEVIEWS
 
+    if modifiers.sessionTableVersion is None or modifiers.sessionTableVersion == SessionTableVersion.AUTO:
+        modifiers.sessionTableVersion = SessionTableVersion.V2
```

Comment:
DON'T MERGE WITHOUT CHANGING THIS TO V1

Reply:
It's set to V2 just to ensure that tests can pass with this set to V2, though I'm also parametrizing the most relevant ones